classes/music.rb

```diff
@@ -0,0 +1,22 @@
+# rubocop:disable Metrics/ParameterLists
```

Comment:
- [ ] Kindly remove the Rubocop disable and fix the error manually as this defies the purpose of using linters in the first place.